Failing live tests

```
================================================================================================================== FAILURES ==================================================================================================================
______________________________________________________________________________________________ LiveSearchQueryTest.test_request_with_condition _______________________________________________________________________________________________

self = <test_live.LiveSearchQueryTest testMethod=test_request_with_condition>

    def test_request_with_condition(self):
        search = SearchQuery("schijf",
                             zip_code="1016LV",
                             distance=100000,
                             price_from=0,
                             price_to=100,
                             condition=Condition.NOT_WORKING,
                             category=category_from_name("Computers en Software"))
    
>       self._validate_response(search)

tests/test_live.py:89: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/test_live.py:44: in _validate_response
    self.assertGreaterEqual(listing.date, datetime.now().date() - timedelta(days=7))
E   AssertionError: datetime.date(2025, 3, 19) not greater than or equal to datetime.date(2025, 3, 20)
========================================================================================================== short test summary info ===========================================================================================================
FAILED tests/test_live.py::LiveSearchQueryTest::test_request_with_condition - AssertionError: datetime.date(2025, 3, 19) not greater than or equal to datetime.date(2025, 3, 20)
```

--- a/tests/test_live.py
+++ b/tests/test_live.py
@@ -12,7 +12,7 @@ class LiveSearchQueryTest(unittest.TestCase):
     haven't deployed breaking changes.
     """
 
-    def _validate_response(self, search):
+    def _validate_response(self, search, check_time: bool = False):
         listings = search.get_listings()
 
         for listing in listings:
@@ -40,8 +40,9 @@ class LiveSearchQueryTest(unittest.TestCase):
 
             # the date object
             self.assertIsInstance(listing.date, date)
-            # should be greater or equal to what we queried for
-            self.assertGreaterEqual(listing.date, datetime.now().date() - timedelta(days=7))
+            if check_time:
+                # should be greater or equal to what we queried for
+                self.assertGreaterEqual(listing.date, datetime.now().date() - timedelta(days=7))
 
             # the full seller object (another request)
             seller = listing.seller.get_seller()
@@ -74,7 +75,7 @@ class LiveSearchQueryTest(unittest.TestCase):
                              offered_since=datetime.now() - timedelta(days=7),
                              category=category_from_name("Fietsen en Brommers"))
 
-        self._validate_response(search)
+        self._validate_response(search, check_time=True)
 
 
     def test_request_with_condition(self):

